static/templates/settings/account-settings.handlebars

```diff
@@ -218,8 +222,13 @@
             </div>
         </div>
         <div id="get_api_key_box">
+            <div class="loading_indicator_spinner"></div>
             <p>{{t "Please re-enter your password to confirm your identity." }}
-              <a href="/accounts/password/reset/" target="_blank">{{t "Never had one? Forgotten it?" }}</a></p>
+                <a class="sea-green" id="api_forgot_password" target="_blank">{{t "Never had one? Forgotten it?" }}</a>
+                <a class="sea-green reset_password_container" id="api_reset_password" style="display: none">{{t "Reset password" }}</a>
+                <a class="sea-green reset_password_container" id="api_resend_link">{{t "Resend" }}</a>
+                <span id="api_email_text">{{t "(Check your email)" }}</span>
```

Comment:
Regarding my remove duplicate comment, you can try using `partials` otherwise left it, for now, Tim will tell whether it's worth to use partials here or not.